test/service/test_app_logging.py

```diff
@@ -43,19 +43,36 @@ def test_request_logs(self):
                     with self.assertLogs(logger=log, level=level) as logs:
                         requests.get(str(url), headers=headers)
                     logs = [(r.levelno, r.getMessage()) for r in logs.records]
-                    headers = {
+                    request_headers = {
                         'host': url.netloc,
                         'user-agent': 'python-requests/2.32.2',
                         'accept-encoding': 'gzip, deflate, br',
                         'accept': '*/*',
                         'connection': 'keep-alive',
                         **headers,
                     }
+                    response_headers = {
+                        'Access-Control-Allow-Origin': '*',
+                        'Access-Control-Allow-Headers': 'Authorization,'
+                                                        'Content-Type,'
+                                                        'X-Amz-Date,'
+                                                        'X-Amz-Security-Token,'
+                                                        'X-Api-Key',
+                        'Content-Security-Policy': f"default-src {sq('self')}",
+                        'Referrer-Policy': 'strict-origin-when-cross-origin',
+                        'Strict-Transport-Security': 'max-age=63072000;'
+                                                     ' includeSubDomains;'
+                                                     ' preload',
+                        'X-Content-Type-Options': 'nosniff',
+                        'X-Frame-Options': 'DENY',
+                        'X-XSS-Protection': '1; mode=block',
+                        'Cache-Control': 'no-store'
+                    }
                     self.assertEqual(logs, [
                         (
                             INFO,
-                            f"Received GET request for '/health/basic', "
-                            f"with {json.dumps({'query': None, 'headers': headers})}."),
+                            "Received GET request for '/health/basic', "
+                            f"with {json.dumps(dict(query=None, headers=request_headers))}."),
```

Comment:
Why was this changed from the `{...}` form to the `dict(...)` form? It takes more characters even without the variable renaming, and doesn't seem to offer any improvement.

Reply:
This change was a requested by Hannes. https://github.com/DataBiosphere/azul/pull/6483#pullrequestreview-2259033681